soft-delete/packages.config

```
@@ -1,15 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net462" />
+  <package id="Azure.Core" version="1.17.0" targetFramework="net462" />
+  <package id="Azure.Storage.Blobs" version="12.10.0-beta.2" targetFramework="net462" />
+  <package id="Azure.Storage.Common" version="12.9.0-beta.2" targetFramework="net462" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net462" />
```

I think Common will be brought in automatically as a dependency. Is the explicit reference needed?
I think Azure.Core is also automatically referenced.

```suggestion
  <package id="Azure.Core" version="1.20.0" targetFramework="net462" />
  <package id="Azure.Storage.Blobs" version="12.10.0" targetFramework="net462" />
  <package id="Azure.Storage.Common" version="12.9.0-beta.2" targetFramework="net462" />
  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net462" />
```

---

Thank you for your advice. I've updated it.